File: DailyFlash Solutions/DailyFlash Solutions/32-DailyFlash_Solutions/19_Feb_Solution_Two/C/prog5.c
#include<stdio.h>

int main() {
	int num;
	printf("Enter num : ");
	scanf("%d", &num);
	int temp = num;
	while(num != 0) {
		printf("%d / %d = %d\n", temp, num % 10, temp / (num % 10));
		num /= 10;
	}
	return 0;
}

File: DailyFlash Solutions/DailyFlash Solutions/32-DailyFlash_Solutions/19_Feb_Solution_Two/Java/prog5.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("num : ");
		int num = sc.nextInt();
		int temp = num;
		while(num != 0) {
			System.out.println(temp + " / " + num % 10 + " = " + temp / (num % 10));
			num /= 10;
		}
	}
}
